documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -472,6 +499,19 @@ Max length: 32 characters
 The `PARTITION` option may be used to specify the boot partition number, if it has not explicitly been set by the `reboot` command (e.g. `sudo reboot N`) or by `boot_partition=N` in `autoboot.txt`.
 This could be used to boot from a rescue partition if the user presses a button.
 
+The latest firmware also allows high partition numbers (> 31) to be overriden. This allows a custom setup of the system hardware watchdog to trigger a reboot with a special high partition number (e.g. 62) which can be detected by the bootloader (using a conditional filter) and remapped to a recovery partition.
```

Comment:
I know it's repeating documentation that appears on another page, but perhaps the wording here talking about "a custom setup of the system hardware watchdog"  also ought to mention the `sudo reboot N` option?

Reply:
I think that's probably unnecessary and repetition tends to be confuusing

Reply:
Oh, I'd totally missed that this page _already_ mentions `sudo reboot N` in just the previous paragraph, sorry! :facepalm:  I must have been tired last night :wink: